src/main/kotlin/app/revanced/meta/ReadmeGenerator.kt

```diff
@@ -0,0 +1,55 @@
+package app.revanced.patches.meta
+
+import java.io.File
+import kotlin.io.writeText
+import kotlin.collections.first
+import app.revanced.patcher.util.patch.implementation.JarPatchBundle
+import app.revanced.patcher.extensions.PatchExtensions.compatiblePackages
+import app.revanced.patcher.extensions.PatchExtensions.patchName
+import app.revanced.patcher.extensions.PatchExtensions.description
+
+class ReadmeGenerator {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
+            //should be moved to a file?
+            val generalReadme =
+                """    
+            # ReVanced Patches
+            🧩 Official patches by ReVanced
+
+            # Patch list
+            """.trimIndent()
+
+            val tableHeader =
+                """
+            |💊Patch|📜Description|🎯Target Package|🏹Target Version|
```

Comment:
```suggestion
            | 💊 Patch | 📜 Description | 🎯 Target Package | 🏹 Target Version |
```

Reply:
It won't affect anything. But for our eyes, I approve of this change.